src/tls_wrap.cc

```diff
@@ -881,6 +889,90 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
 
 
+#ifdef OPENSSL_PSK_SUPPORT
+
+unsigned int TLSWrap::PskServerCallback(SSL *ssl,
```

Comment:
Actually, not sure why I couldn't get it implemented in `SecureContext` before. I just took another shot and it works fine. Seems even better?